identity/ideth/service.go

```diff
@@ -430,7 +456,8 @@ func (i service) AddKeysForAccount(acc config.Account) error {
 func (i service) ValidateSignature(signature *coredocumentpb.Signature, message []byte) error {
 	centID := id.NewDIDFromBytes(signature.EntityId)
 
-	err := i.ValidateKey(context.Background(), centID, signature.PublicKey, &(id.KeyPurposeSigning.Value))
+	sigTime := time.Unix(signature.Timestamp.Seconds, int64(signature.Timestamp.Nanos))
```

Comment:
lets not use the signature timestamp for this. As a bad actor, I can still change the time stamp in the signature since timestamp is not included in the signature. I would pass this as nil until we have the timestamp in the coredocument

Reply:
but what does bad actor gain out of that? Since signature is not part of any root, anything can be faked here

Reply:
he can fake the validation of the signature since we are relying on the timestamp to validate when key is revoked, right?

Reply:
yes, but any validation here is done by the bad actor here, so he can just say that signature is not valid without even trying to validate. 
@lucasvo Is this something we should consider?

Reply:
@lucasvo I think we confused two different validations here, I had a discussion with @manuelpolzhofer @vedhavyas on this and its clear to me that I was just talking about p2p signature validation vs you guys talking about NFT level key revoked validations.